### PR TITLE
Content path and * wildcard documentation

### PR DESCRIPTION
**Pull request by simonswiss**
Improves and extends the documentation of the `path` option on singletons and collections.

Fixes https://github.com/Thinkmill/keystatic/issues/529

### CODE REVIEW

File: docs/src/content/pages/how-keystatic-organises-your-content.mdoc [hunk 1]
```diff
@@ -35,33 +35,84 @@ export default config({
...
+  ├── en  
+      └── post-1.mdoc
+  ├── fr  
+      └── post-1.mdoc
+```
+
+You can enable this by using the `**/*` wildcard in your `path`: 
+
+```
+path: 'content/posts/**/*'
```
Comment (emmatown) [inline docs/src/content/pages/how-keystatic-organises-your-content.mdoc:96]:
````suggestion
You can enable this by using the `**` wildcard in your `path`: 

```
path: 'content/posts/**'
````

Comment (simonswiss) [reply docs/src/content/pages/how-keystatic-organises-your-content.mdoc]: Updated these 2 + another instance of `**/*` 👍

**Review by emmatown — APPROVED** [review]
This kinda confused me when I was trying to review it because I did not expect it to be under "Organised content" but having this documented somewhere seems better than the status quo so 🚢

**Review by JedWatson — APPROVED** [review]

**Review by JedWatson — APPROVED** [review]
Approved. Agree this would make more sense somewhere else, but we can change that in a future PR, this is great new content! thanks @simonswiss

**Comment by simonswiss** [comment]
Thank you!

And yep, I agree this probably needs a dedicated page or API reference thing, rather than a core "concept".

**Pull request merged**